### PR TITLE
Zarina posting about accessible PDF

### PR DESCRIPTION
**Pull request by zmustapha**

### CODE REVIEW

File: content/articles/accessible-pdf-wheel.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+Identities that have the strongest effect on how you perceive yourself; (5)
+Identities that have the greatest effect on how others perceive you.</p>
+
+{{< figure
+    src="/img/assets/identity-wheel.png"
+    alt="The Identity Wheel Activity" >}}
+
+Each box on the wheel represents a different facet of identity. The learner
+will type in the textboxes the number that corresponds with each reflective
+prompts in the center of the circle as it relates to a particular identity
```
Comment (nbuonin) [inline content/articles/accessible-pdf-wheel.md]: Would it be `each reflective prompt`?

File: content/articles/accessible-pdf-wheel.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+facet. An interactive PDF seemed like a good fit for Identity Wheel worksheet
+because the learner can work on the activity on- or offline, save it or print
+it for future reference. The worksheet just needed to be made accessible;
+exercise didn’t look so complex.
```
Comment (nbuonin) [inline content/articles/accessible-pdf-wheel.md]: Maybe add `the exercise`.

File: content/articles/accessible-pdf-wheel.md [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+
+## Solution: first UX, then UX, then tools, then more UX
+
+The goal of the Identity Wheel exercise is to get learners to associate the
+reflective prompts to the twelve identity facets. The initial visual layout and
```
Comment (nbuonin) [inline content/articles/accessible-pdf-wheel.md]: Maybe `associate... with...` rather than `to`

File: content/articles/accessible-pdf-wheel.md [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+__Final result: [Accessible interactive Identity Wheel Activity PDF](/img/assets/identity_wheel_compiled.pdf)__
+
+## Lessons learned
+
+What I’ve learned from this whole process are:
+
+1. Accessibility experience __is user experience__ and __is learning
+experience__. Get these right first, and the content will be accessible.
+2. PDF content needs to be in a logical, or meaningful structure and sequential
+order using semantic structure in the original document as much as possible
```
Comment (nbuonin) [inline content/articles/accessible-pdf-wheel.md]: `PDF content... before exporting to PDF` doesn't quite gel. This clause is confusing too: `sequential order using semantic structure`. How about: `Content needs to be semantically structured and sequentially ordered before exporting to PDF`?

File: content/articles/accessible-pdf-wheel.md [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+
+{{< figure
+    src="/img/assets/two-col-text.png"
+    alt="The paragraph of text in a two-column layout. A sighted reader perceives the content flow as top-down, line by line, column by column." >}}
+
+A screen reader, on the other hand, doesn’t recognize the content structure as
+hinted by the columns because there’s nothing in the inaccessible PDF that
+passes that structure along. The screen reader reads the inaccessible PDF from
+top to bottom, left to right, across columns. The following video demonstrates
+the nonsensical screen reading of an inaccessible PDF content.
```
Comment (nbuonin) [inline content/articles/accessible-pdf-wheel.md]: Drop `an`

File: content/articles/accessible-pdf-wheel.md [hunk 6]
```diff
@@ -0,0 +1,229 @@
...
+Identities that have the greatest effect on how others perceive you.</p>
+
+{{< figure
+    src="/img/assets/identity-wheel.png"
+    alt="The Identity Wheel Activity" >}}
+
+Each box on the wheel represents a different facet of identity. The learner
+will type in the textboxes the number that corresponds with each reflective
+prompts in the center of the circle as it relates to a particular identity
+facet. An interactive PDF seemed like a good fit for Identity Wheel worksheet
```
Comment (nbuonin) [inline content/articles/accessible-pdf-wheel.md]: Add `the`; `good fit for the Identity Wheel`

File: content/articles/accessible-pdf-wheel.md (hunk 3 above)
Comment (nbuonin) [reply content/articles/accessible-pdf-wheel.md]: It's funny, when I hear it with VoiceOver it sounds natural. It's ambiguous.

File: content/articles/accessible-pdf-wheel.md [hunk 7]
```diff
@@ -0,0 +1,229 @@
...
+### 1. Design: Adobe InDesign
+
+PDF accessibility begins from the original source, and since the worksheet has
+a particular visual layout, I used Adobe InDesign to create the source
+document. I designed the worksheet according to the accessibility experience
+structure and order previously described. The key to the correct reading and
+tabbing order in Adobe InDesign is the Articles Panel, and Tag Panel for
+structure—headings, paragraphs, images. I made sure that the source document
+had no typos or errors, and that it complied to the accessibility experience
+_before exporting to interactive PDF_ to minimize repairs in subsequent steps.
```
Comment (nbuonin) [inline content/articles/accessible-pdf-wheel.md]: Add `an`: `before exporting to an interactive`.

File: content/articles/accessible-pdf-wheel.md (hunk 1 above)
Comment (zmustapha) [reply content/articles/accessible-pdf-wheel.md]: Yes it is. Thank you

File: content/articles/accessible-pdf-wheel.md (hunk 2 above)
Comment (zmustapha) [reply content/articles/accessible-pdf-wheel.md]: Indeed. Thank you

File: content/articles/accessible-pdf-wheel.md (hunk 3 above)
Comment (zmustapha) [reply content/articles/accessible-pdf-wheel.md]: All good. will substitute with `with`

File: content/articles/accessible-pdf-wheel.md (hunk 4 above)
Comment (zmustapha) [reply content/articles/accessible-pdf-wheel.md]: Thanks

File: content/articles/accessible-pdf-wheel.md (hunk 6 above)
Comment (zmustapha) [reply content/articles/accessible-pdf-wheel.md]: Thanks

File: content/articles/accessible-pdf-wheel.md (hunk 7 above)
Comment (zmustapha) [reply content/articles/accessible-pdf-wheel.md]: Thanks

**Pull request merged**